### ISSUE TITLE
Windows compatibility problem

### ISSUE THREAD

**Issue by kriek** [reverbc/pylint-pytest#1]
Hi,

when trying to use pylint-pytest==0.2.0 plugin from Windows, I have the following error:
```
INTERNALERROR>   File "d:\programs\python38\lib\site-packages\pylint_pytest\__init__.py", line 24, in register
INTERNALERROR>     checker = importlib.import_module(module, package=os.path.basename(dirname))
INTERNALERROR>   File "d:\programs\python38\lib\importlib\__init__.py", line 127, in import_module
INTERNALERROR>     return _bootstrap._gcd_import(name[level:], package, level)
INTERNALERROR>   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
INTERNALERROR>   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
INTERNALERROR>   File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
INTERNALERROR> ModuleNotFoundError: No module named '\\checkers\\class_attr_loader'
```

Updating from `module = module.replace('/', '.')` to `module = module.replace(os.sep, '.')` would fix the portability issue.

**Comment by reverbc** [comment]
Thanks for reporting this! I'll fix it and set up a WIndows CI to catch such silly issues 🤦

**Comment by skshetry** [comment]
Hi, @reverbc, I was wondering if you could make a release with this change? Thanks.

**Comment by reverbc** [comment]
@skshetry thanks for the nudge! Just released v0.3.0 to pypi 🙇🏻‍♂️
